package.json

```diff
@@ -26,14 +26,16 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "bpk-tokens": "^27.4.7",
+    "bpk-tokens": "^32.0.2",
     "lodash": "^4.17.15",
     "tinycolor2": "^1.4.1"
   },
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
```

Comment:
Added liniting libs here as requiring `eslint-config-skyscanner` causes a circular dependency and means we cannot run any linting or prettier tools